docs/packages/pkg/Redis++.rst

```diff
@@ -0,0 +1,29 @@
+.. spelling::
+
+    Redis++
+    redis
+
+.. index:: Redis++
+
+.. _pkg.Redis++:
+
+Redis++
+=======
+
+-  `Official <https://github.com/sewenew/redis-plus-plus/>`__
+-  `Example
+   CMakeLists.txt <https://github.com/cpp-pm/hunter/blob/master/examples/Redis++/CMakeLists.txt>`__
+-  Added by `Ronny Nowak <https://github.com/Dargun>`__
+
+.. code-block:: cmake
+
+    hunter_add_package(HiRedis)
```

Comment:
is it really needed to add and find the packages to build and use redis++? The dependencies should be found by Redis++ itself

Reply:
no redis++ is not managing that so we need to install libuv and hiredis first and then we can tell rediss++ 
with CMAKE_PREFIX_PATH=${HUNTER_INSTALL_PREFIX} where he can find installed packages.